core/services/chainlink/config_test.go

```diff
@@ -1510,7 +1510,13 @@ func TestConfig_Validate(t *testing.T) {
 		- 1: 2 errors:
 			- ChainID: missing: required for all chains
 			- Nodes: missing: must have at least one node
-	- Aptos.0.Enabled: invalid value (1): expected bool`},
+	- Aptos: 4 errors:
+		- Enabled: invalid value (1): expected bool
+		- ChainID: invalid value (): expected string
```

Comment:
Should this one be a "missing" error?
```suggestion
		- ChainID: invalid value (): expected string
```